frontend/src/routes/(authenticated)/project/[project_code]/+page.svelte

```diff
@@ -75,120 +80,149 @@
   }
 
   $: userId = user.id;
-  $: canManage = isAdmin(user) || user.projects.find(p => p.code == project?.code)?.role == 'Manager';
+  $: canManage = isAdmin(user) || user.projects.find((p) => p.code == project?.code)?.role == 'Manager';
 
   const projectNameValidation = z.string().min(1, $t('project_page.project_name_empty_error'));
+
+  let deleteProjectModal: ConfirmDeleteModal;
+
+  async function softDeleteProject(): Promise<void> {
+    const result = await deleteProjectModal.open(_project.name, async () => {
+      const { error } = await _deleteProject(_project.id);
+      return error?.message;
+    });
+    if (result.response === DialogResponse.Submit) {
+      notifyWarning($t('delete_project_modal.success', { name: _project.name, code: _project.code }));
+      await goto(data.home);
```

Comment:
instead of this can we use `history.back()`? then they just go back to where they were?

Reply:
No, that seems really weird to me. They could land anywhere, even on a different site.

Reply:
that's true, that would be a bit weird.